modules/economy.js

```diff
@@ -1,293 +1,235 @@
+          if (err) client.logError(err)
+          if (err) throw new Error('DB_ERROR')
+          module.exports.getMemberInventoryAndBalance(client, member, guild, callback)
+        })
       }
     })
     EfM.finish()
   },
   getLeaderboard: (client, message) => {
 
   },
-  makeMoneyTransferToUser: (client, guild, fromUser, toUser, amount, callback) => {
-    client.pool.query('SELECT * FROM `guildEconomyUserBank` WHERE guild = ? AND member = ?', [guild.id, fromUser.id], (err, fromUserResult) => {
-      if (err) client.logError(err)
-      if (fromUserResult && Object.prototype.hasOwnProperty.call(fromUserResult, 0)) {
-        if (parseInt(fromUserResult[0].amount) >= amount) {
-          client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(fromUserResult[0].amount) - amount), fromUser.id, guild.id], (err) => {
-            if (err) client.logError(err)
-          })
-          client.pool.query('SELECT * FROM `guildEconomyUserBank` WHERE guild = ? AND member = ?', [guild.id, toUser.id], (err, toUserResult) => {
-            if (err) client.logError(err)
-            if (fromUserResult && Object.prototype.hasOwnProperty.call(fromUserResult, 0)) {
-              client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(toUserResult[0].amount) + amount), toUser.id, guild.id], (err) => {
-                if (err) client.logError(err)
-              })
-            } else {
-              client.pool.query('INSERT INTO `guildEconomyUserBank` (`member`, `guild`, `amount`) VALUES (?, ?, ?)', [toUserResult.id, guild.id, amount], (err) => {
-                if (err) client.logError(err)
-              })
-            }
-          })
-          if (callback) callback(true)
-        } else {
-          if (callback) callback(false)
-        }
+  makeMoneyTransferToUser: (client, guild, fromUser, toUser, moneyToTransfer, callback) => {
+    module.exports.getMemberInventoryAndBalance(client, fromUser, guild, (fromUserInventoryAndBalance) => {
+      if (fromUserInventoryAndBalance.amount >= moneyToTransfer) {
+        module.exports.getMemberInventoryAndBalance(client, toUser, guild, (toUserInventoryAndBalance) => {
+          try {
+            module.exports.updateMemberBalance(client, fromUser, guild, (parseInt(fromUserInventoryAndBalance.amount) - moneyToTransfer))
+            module.exports.updateMemberBalance(client, fromUser, guild, (parseInt(toUserInventoryAndBalance.amount) + moneyToTransfer))
+          } catch (err) {
+            client.logError(err)
+          }
+        })
       } else {
-        module.exports.getDailyMoney(client, fromUser, guild, false)
-        module.exports.makeMoneyTransferToUser(client, guild, fromUser, toUser, amount, callback)
+        throw new Error('ECO_XX01')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
        throw new Error("ECO_XX01")
```